cypress/e2e/pages/vaults/[vaultid]/index.spec.ts

```diff
@@ -244,15 +249,19 @@ context("/vaults/[vaultid] on mobile", () => {
 
     it("should have collateral details card", () => {
       cy.findByTestId("CollateralDetailsMobile.Cards").within(() => {
-        cy.findAllByTestId("CollateralCard").within(() => {
-          cy.findByTestId("CollateralCard.AssetIcon").should("be.visible");
-          cy.findByTestId("CollateralCard.displaySymbol").should("be.visible");
-          cy.findByTestId("CollateralCard.CollateralAmountTitle").should(
-            "be.visible"
-          );
-          cy.findByTestId("CollateralCard.CollateralAmount").should(
-            "be.visible"
-          );
+        cy.findAllByTestId("CollateralCard").each(($el) => {
+          cy.wrap($el).within(() => {
+            cy.findByTestId("CollateralCard.AssetIcon").should("be.visible");
+            cy.findByTestId("CollateralCard.displaySymbol").should(
+              "be.visible"
+            );
+            cy.findByTestId("CollateralCard.CollateralAmountTitle").should(
+              "be.visible"
+            );
+            cy.findByTestId("CollateralCard.CollateralAmount").should(
+              "be.visible"
```

Comment:
Can we make line 255, 258 & 261 into single line like this cy.findByTestId("CollateralCard.AssetIcon").should("be.visible");
Just to maintain its readability